tools/doxygen/Doxyfile.c++

```diff
@@ -876,59 +876,6 @@ include/grpc++/support/string_ref.h \
 include/grpc++/support/stub_options.h \
 include/grpc++/support/sync_stream.h \
 include/grpc++/support/time.h \
-include/grpc/byte_buffer.h \
```

Comment:
note to self: I'll need to investigate why some of the files seem to be missing from the doxyfiles.

Reply:
Resolved. The doxyfiles weren't including public header files transitively, so I fixed that.